File: logger.java
import okhttp3.*;

enum LogType {
    OK,
    INFO,
    WARN,
    ERROR
}

public class Main {
    private static final String loggerID = "example";

    public static void main(String[] args) {
        logger("Let the logging begin!");
        System.out.println("Logging successful.");
    }

    public static void logger(String message, LogType logType) {
        String serverURL = "https://hlog.deno.dev";
        String url = serverURL + "/" + loggerID + "/" + logType.name().toLowerCase();

        OkHttpClient client = new OkHttpClient();

        MediaType mediaType = MediaType.parse("text/plain");
        RequestBody requestBody = RequestBody.create(mediaType, message);

        Request request = new Request.Builder()
                .url(url)
                .post(requestBody)
                .build();

        try (Response response = client.newCall(request).execute()) {
            // Ignore the response
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

    public static void logger(String message) {
        logger(message, LogType.INFO);
    }
}

File: logger.kt
import okhttp3.MediaType.Companion.toMediaType
import okhttp3.OkHttpClient
import okhttp3.Request
import okhttp3.RequestBody.Companion.toRequestBody

const val loggerID = "example"

enum class LogType {
    OK,
    INFO,
    WARN,
    ERROR
}

fun logger(message: String, logType: LogType = LogType.INFO) {
    val serverURL = "https://hlog.deno.dev"
    val url = "$serverURL/$loggerID/${logType.name.toLowerCase()}"
    val client = OkHttpClient()

    val requestBody = message.toRequestBody("text/plain".toMediaType())
    val request = Request.Builder()
        .url(url)
        .post(requestBody)
        .build()

    client.newCall(request).execute().use { response ->
        // Ignore the response
    }
}

fun main() {
    logger("Let the logging begin!")
    println("Logging successful.")
}
